Python/formatter_unicode.c

```diff
@@ -280,7 +281,10 @@ parse_internal_render_format_spec(PyObject *format_spec,
 
     if (end-pos > 1) {
         /* More than one char remain, invalid format specifier. */
-        PyErr_Format(PyExc_ValueError, "Invalid format specifier");
+        PyErr_Format(PyExc_ValueError,
+                     "Invalid format specifier: '%s' for object of type '%s'",
```

Comment:
For consistency with other errors in this file, you should remove the colon after "specifier".

Also, as a rule the code limits type name to 200 chars, so change "type '%s'" to "type '.200s.".

Reply:
Replaced.